src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -60,12 +61,12 @@ public function fillable(array $fillable)
     /**
      * Merge new fillable attributes with existing fillable attributes on the model.
      *
-     * @param  array  $fillable
+     * @param  array|string  $fillable
```

Comment:
I added this for the IDE to check if the inserted variable is an array or string, but will be checked later